tests/drivers/build_all/app.overlay

```diff
@@ -116,6 +116,15 @@
 			#include "uart.dtsi"
 		};
 
+		hx711 {
+        	compatible = "avia,hx711";
```

Comment:
Wrong indentation. Mixed tabs + spaces, and probably one more tab is needed.

Reply:
Right.

Reply:
Fixed.